flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -70,11 +74,14 @@ public static Builder forRowData() {
    * Source builder to build {@link DataStream}.
    */
   public static class Builder {
+    private static final Set<String> LOCALITY_WHITELIST_FS = ImmutableSet.of("hdfs");
```

Comment:
we need a better name for the constant. maybe `FILE_SYSTEM_SUPPORT_LOCALITY`?

Reply:
Modified along with the naming in spark.